packages/firestore/src/remote/remote_event.ts

```diff
@@ -49,6 +56,30 @@ export class RemoteEvent {
      */
     readonly resolvedLimboDocuments: DocumentKeySet
   ) {}
+
+  /**
+   * Creates a synthesized RemoteEvent that can be used to apply a CURRENT
+   * status change for a query that is executed by a different tab.
+   */
+  // PORTING NOTE: Multi-tab only
+  static createSynthesizedRemoteEventForSyncStateChange(
+    targetId: TargetId,
+    current: boolean
```

Comment:
Can we avoid mixing current and syncState terminology and pick one? (probably 'current' since that's what you use as the parameter name and in the method description)

Reply:
As discussed via Doodle, I updated this to use "forCurrentChange". I originally avoided this name because it is kind of ambiguous to non-Firestore folks.